HOWTO_meta_quest.md

```diff
@@ -235,71 +256,60 @@ More capture options can be found in the
-  4. Make sure "app-openxr-vulkan"
-
-    is selected 
-  5. Press the "Play" triangle to run
+### 7. Install and Run the Application
 
-    a. You may have to first select to enable commands from the host
-    system on the headset
+Install the application as normal and run it.
+You may have to stop and uninstall any older 
 
-This shows that you can run the application on the target.
+Then run the application as normal to capture
+the content.
 
-### 5. Run the Application
-
-Repeat the steps from section 2. above.
-
-This time, however, you should see a generated capture file after
-exiting the application in the `/sdcard/Download` folder called
-`openxr_capture_frames_1_to_500.gfxr` since you have selected to save
-only some of the frames.
-
-### 6. Verify the capture file
-
-Check that the above file exists:
+If the application fails to launch, review the
+`adb logcat` information for any fatal or error messages.
+Here is a useful filter for logcat messages.
 
 ```bash
-adb shell ls /sdcard/Download/openxr_capture*.gfxr
+adb logcat -s DEBUG vulkan VulkanLoader VulkanLoaderAndroid OpenXR OpenXR-Loader gfxrecon
 ```
 
-### 7. Disable the capture layer
+Once complete, if everything executed correctly, the capture
+file should be present in the `/sdcard/Download` folder.
 
-Disable the capture layer globally and also restore the Vulkan usage of
-the HWUI:
+Check that it exists:
 
 ```bash
-adb shell "setprop debug.vulkan.layers ''"
-adb shell "settings put global gpu_debug_app ''"
-adb shell "settings put global gpu_debug_layers ''"
+adb shell ls /sdcard/Download/openxr_capture*.gfxr
 ```
 
-## Replaying the IGL Content
+## Replaying a Capture File
+
 
+### 1. Install the Quest Replay Application
 
-### 1. Install the replay application
+Install the GFXReconstruct Quest Replay application
+that is built as part of the Android build of
+GFXReconstruct using the `gfxrecon.py` script:
 
-Now, we need to install the Replay application that we built as part of
-the GFXReconstruct source.  Install the replay APK from the root of the
-built source tree by using the `gfxrecon.py` script:
+For example, if installing the Debug build of
+the Quest replay app:
 
 ```bash
+cd {gfxreconstruct_root}
 python3 android/scripts/gfxrecon.py install-apk android/tools/quest_replay/build/outputs/apk/debug/quest_replay-debug.apk
 ```
 
+Replacing `{gfxreconstruct_root}` with the location of your
+GFXReconstruct source pulled down from Github above in
+the [Building GFXReconstruct](#building-gfxreconstruct)
+section.
+
 ### 2. Force External Storage Permissions
 
 Even though the Quest Replay application is built enabling external
-storage access permissions in its Android Manifest file, it doesn't appear
-to always work. Because of this, just force on the permission right
-before attempting to run and access the files in the `/sdcard/downlaod`
+storage access permissions in its Android Manifest file,
+it doesn't always appear to stick.
```

Comment:
maybe instead say that it needs to be specified each time the apk is installed (that was my experience)